junit-runtime/src/main/scala/org/junit/ComparisonFailure.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Ported from https://github.com/junit-team/junit
+ */
+package org.junit
+
+object ComparisonFailure {
+  private final val MAX_CONTEXT_LENGTH: Int = 20
```

Comment:
The Scala spec. Without type ascription is becomes a *constant expression*, which can be folded.